### PR TITLE
Friends Settings-UI

### PR DESCRIPTION
**Pull request by lennet**
Bildschirmfoto 2019-09-23 um 23 49 20
Bildschirmfoto 2019-09-23 um 23 51 19
Bildschirmfoto 2019-09-23 um 23 56 15

### CODE REVIEW

**Comment by headione** [comment]
is it possible to also remove friends? 🤔

**Comment by lennet** [comment]
Good point forgot to add that. Added a swipe to delete gesture 
Bildschirmfoto 2019-09-30 um 12 50 44

File: CriticalMass/TypeNameProtocol.swift [hunk 1]
```diff
@@ -24,3 +24,4 @@ public extension TypeNameProtocol {
 }
 
 extension UIView: TypeNameProtocol {}
+extension UIViewController: TypeNameProtocol {}
```
Comment (mltbnz) [inline CriticalMass/TypeNameProtocol.swift]: Oh. Thought we had that already. Nice

File: CriticalMass/FollowFriendsViewController.swift [hunk 2]
```diff
@@ -9,37 +9,62 @@ import UIKit
...
+    }
+
+    private func configureNavigationBar() {
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .action, target: self, action: #selector(shareLinkButtonTapped))
     }
 
     private func configureQRCodeView() {
-        let view = QRCodeView()
+        let backgroundView = QRCodeBackgroundView(frame: CGRect(origin: .zero, size: CGSize(width: 280, height: 280)))
+        backgroundView.center = self.view.center
```
Comment (mltbnz) [inline CriticalMass/FollowFriendsViewController.swift]: I'd prefer some NSLayoutConstraint but this is fine too

File: CriticalMass/FollowFriendsViewController.swift [hunk 3]
```diff
@@ -9,37 +9,62 @@ import UIKit
...
-        button.setTitle("Share Link", for: .normal)
-        button.setTitleColor(.blue, for: .normal)
-        button.addTarget(self, action: #selector(shareLinkButtonTapped), for: .touchUpInside)
-        button.sizeToFit()
-        button.center = view.center.applying(CGAffineTransform(translationX: 0, y: -200))
-        view.addSubview(button)
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        UIScreen.main.brightness = 100
```
Comment (mltbnz) [inline CriticalMass/FollowFriendsViewController.swift]: 😎

File: CriticalMass/FollowFriendsViewController.swift (hunk 2 above)
Comment (lennet) [reply CriticalMass/FollowFriendsViewController.swift]: Good point. Added an helper function for that

**Pull request merged**